packages/cli/src/commands/server/server.ts

```diff
@@ -106,5 +106,9 @@ export default {
       description: 'Path to the CLI configuration file',
       parse: (val: string) => path.resolve(val),
     },
+    {
+      name: '--interactive',
+      description: 'Disables interactive mode',
```

Comment:
I think something is wrong with the name here. How am I supposed to disable it? `--interactive false`? We can either rephrase it so that its presence negates (e.g. `--no-interactive`) or leave it as is and add `default: true` as well as `description: Enables interactive mode, true by default`.

Reply:
This was @thymikee's idea:
> I'd prefer to have --interactive flag and use a built-in feature of commanderjs that let us negate the fag with no- prefix, so --no-interactive would disable it then.